File: nodejs/7 kyu/69.js
// https://www.codewars.com/kata/5b5604e26dc79e6832000101/train/javascript

// Passed

function invertHash(hash) {
  const keys = Object.keys(hash);
  let invertedHash = new Object();
  keys.forEach((key) => {
    invertedHash[hash[key]] = key;
  });

  return invertedHash;
}

const output = invertHash({ a: "1", b: "2", c: "3" });
console.log(output);

File: python/7 kyu/69.py
# https://www.codewars.com/kata/590f5b4a7bbb3e246000007d/train/python

# Passed

def comes_after(st, l):
    result = str()
    while st:
        if l.lower() not in st.lower():
            break # no more occurences of l
        
        after_idx = st.lower().find(l.lower()) + 1
        if after_idx == len(st):
            # no characters after letter
            break

        result += st[after_idx] if st[after_idx].isalpha() else str()
        st = st[after_idx:]        

    return result

output = comes_after("Free coffee for all office workers!", 'F')
print(output)